### ISSUE TITLE
Make tool for dependency analysis

### ISSUE THREAD

**Issue by jaekwon** [tendermint/tendermint#1208]
Please make a simple Makefile recipe that produces an analysis (ideally, but not necessarily a graphical image with arrows pointing to dependencies) of dependency binary size?

One way would be to run the procedure on the following link:

https://hackernoon.com/a-story-of-a-fat-go-binary-20edc6549b97

**Comment by jaekwon** [comment]
If I tag something with bounty, it means I'll pay for someone to solve it (while it needs to be solved).  AIB employees can participate.

**Comment by use-n-delete** [comment]
Hi !

I prepared a pull request with minimalistic variant of tool that you described. Here is the link: https://github.com/tendermint/tendermint/pull/1218 I am not sure if the develop branch should be PR target. I am waiting for your feedback.

**Comment by jaekwon** [comment]
Just saw this, I will take a look in the coming days.  By the end of the weekend by latest.  Thank you!

**Comment by use-n-delete** [comment]
Okay, take your time :)

**Comment by use-n-delete** [comment]
Hi again,

I noticed that my pull request has been accepted and merged. You can tip me with whatever you think is appropriate:

0xd3abe4cB11Ddf122ae6aFBbfFd6FAcfe68CCe6FC (ETH address)

For future reference, you can consider using https://www.gitcoin.co/ if you want to outsource simple, well-defined tasks (like this one). It's a rather nascent platform, but I think it can speed up completion of bounty-like tasks for Cosmos/Tendermint.

I am really excited about your project and I keep my fingers crossed for successful launch of Cosmos. Good luck !

**Comment by jaekwon** [comment]
Thank you @use-n-delete, we're looking into payment procedure.
Are you OK with going through a KYC procedure for our corporate records?  It's hard to pay out bounty payments without this.  I'll make sure it's worth your time though.

**Comment by use-n-delete** [comment]
Yes, I am okay with it. Please instruct me what I should do to claim this bounty.
